app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,18 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
+  end
+
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @like = Like.new(user: @user, post: @post)
+
+    if @like.save
+      @post.increment!(:likes_counter)
```

Comment:
[OPTIONAL]
- [ ] Kindly remove this line of code from the controller because the `likes_counter` column is automatically increment whenever a like is created thanks to the `after_create :increment_post_likes_counter` code in the `Like` model

Reply:
I really appreciate your review. Thanks alot 💯